tests/e2e/tests/electron/critical-path/database/add-ssh-db.e2e.ts

```diff
@@ -131,6 +131,8 @@ test
     };
     // Verify that user can add SSH tunnel with Password for OSS Cluster database
     await myRedisDatabasePage.AddRedisDatabaseDialog.addStandaloneSSHDatabase(sshDbClusterPass, sshWithPass);
+    // should be deleted after https://redislabs.atlassian.net/browse/RI-5995
```

Comment:
could you mark it as with 'TODO' text in this comment to find it in future easier?